CarbonFight/actions
# feat: add more test for action (create/delete) + jsdoc + mock `updateStats` + unit testing setup for `updateStats` function

- First schema validation using Zod
- Comment breaking function `updateStats()`
- Many utils to help unit testing
- jsDoc for all utils
- Testing actions:
  - update
  - delete
  - create

## How to test:
- run `just test`
- Logs things in create/update/delete to see if everything is doing well

## Warning 

Tests are failing inside `update` because of code inside the firebase function. This code should be fixed but it's unrelated to this PR.
